iOS/PixelDefinitions/pixels/vpn_events.json5

```diff
@@ -104,16 +103,15 @@
             },
             // Per-step underlying error domains
             {
-                "keyPattern": "feature\\.data\\.ext\\.(controller_start|oauth|tunnel_start)_error\\.underlying_domain[0-9]*",
+                "keyPattern": "feature\\.data\\.ext\\.(controller_start|oauth|tunnel_start)_error\\.underlying_domain[0-9]?",
```

Comment:
In practice, this should be fine - our pixel library can go infinitely deep with underlying errors, but usually it's <= 3 and I've never seen anything higher than 5. If this does end up tripping the pixel validator I can't imagine we'd cut this off at the pixel library level since having the error information is vital, but we can have that conversation if it happens.